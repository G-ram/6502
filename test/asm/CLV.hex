a9 ff 69 02 8d 00 01 a9 7f 69 02 8d 01 01 b8 
