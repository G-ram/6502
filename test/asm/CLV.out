A: 82
X: 00
Y: 00
P: 10010000
S: 20
PC: 60f
100 : 01
101 : 82
